store/transfer_processor.go

```diff
@@ -294,3 +300,17 @@ func (s *transferProcessor) process(ctx context.Context, events []*model.Transfe
 	}
 	return nil
 }
+
+// Get last recorded balance data for these accounts
+func (s *transferProcessor) getLatestBalances(contractAddr common.Address, addrs [][]byte) (map[common.Address]*model.Account, error) {
+	balances, err := s.accountStore.FindLatestAccounts(contractAddr, addrs)
+	if err != nil {
+		s.logger.Error("Failed to get previous balances", "len", len(addrs), "err", err)
```

Comment:
It's seem to me that this line can be removed, because you have already logged message in outside. https://github.com/getamis/eth-indexer/pull/42/commits/ad0459e512a2309ea7046f114df8a4cf9c0b219b#diff-778fd13deb3062e86beaee58b3afff70R196

Reply:
thanks! done